lib/UserManagement.php

```diff
@@ -257,6 +257,9 @@ public function listOrganizationMemberships(
         $params = [
             "organization_id" => $organizationId,
             "user_id" => $userId,
+            "limit" => $limit,
```

Comment:
not related but I realized I missed those in #190.
I can extract to another PR if you prefer.

Reply:
no worries, good catch

Reply:
oh i just noticed the return type in the comment of this method is inaccurate; can be fixed here or in a followup, whatever you prefer

Reply:
good catch too 😄

Reply:
🙌